tests/test_hs_restclient.py

```diff
@@ -313,6 +312,128 @@ def test_folder_delete(self):
 
         self.assertEqual(response['resource_id'], '511debf8858a4ea081f78d66870da76c')
         self.assertEqual(response['path'], 'model/initial')
-                
+
+
+class TestResourceCopy(unittest.TestCase):
+    @with_httmock(mocks.hydroshare.resourceCopy_post)
+    def test_resource_copy(self):
+        hs = HydroShare()
+        response = hs.resource('511debf8858a4ea081f78d66870da76c').copy()
```

Comment:
@aphelionz  I assume this line shows how the user will be using the copy api. I am wondering if this will introduce  inconsistency in api calling pattern.  For example, for creating a resource, we have ```hs.createResource()```. Based on that api pattern I think user would expect the copy api to be ```hs.copyResource()```.  @alvacouch your thoughts on this?